queries/fusion/locals.scm

```diff
@@ -1,3 +1,21 @@
+;; Fusion base
```

Comment:
These are all 'blind' definitions. I do not know how to test a resulting behavior, so i can validate if these definitions are right.
Do you have any advice or hint how to test or do any qa?

Reply:
you can create some tests similar to https://github.com/theHamsta/nvim-treesitter/blob/2bf58527049185347308300dff5184871bd99764/tests/query/highlights_spec.lua#L26

 Theses could be general query tests where you just check here https://github.com/theHamsta/nvim-treesitter/blob/2bf58527049185347308300dff5184871bd99764/tests/query/highlights_spec.lua#L61-L69 whether assertion == capture . If you want to go more fancy you could even call `locals.find_definition(node_at_point, bufnr)` to find the definition node and check whether it matches the assertion (e.g. use `reference.1` on all the references and `definition.1` on the definition.

Reply:
Thanx for the clarification.
this sounds like a good option. I will give it a try. When I have time I will make a lua crash course and give it a try.

great review as always.
all the best and a happy new year

Reply:
Happy new year and thanks for the contribution! If you want to I can also attach this feature to the injection tests that got some how stuck in review as it does a similar task.